ckanext/cfpb_extrafields/validators.py

```diff
@@ -6,11 +6,6 @@ def Invalid(message):
     import ckan.plugins.toolkit as tk
     raise tk.Invalid(message)
 
-def required_field(value):
```

Comment:
thanks for the review, Dan! Yup, it was a little hidden away but there was, indeed, a 'not_empty'